### ISSUE TITLE
Add unit tests

### ISSUE THREAD

**Issue by jasoncouture** [Microsoft/Docker.DotNet#176]
I would like to add unit tests as described in readme.md
I propose NUnit. If that is okay, I will fork and get started as soon as I can.

**Comment by jterry75** [comment]
Last I checked the only one that supported .NET Core was xUnit. Can we do that instead?

**Comment by JosephWoodward** [comment]
I agree on going with xunit, it's the framework supported out of the box with the .NET Core CLI too. It took NUnit a long time to add .NET Core support too which knocked my confidence in it keeping up with changes, where as xunit were on top of it.

**Comment by jasoncouture** [comment]
Sure, I can do that. I've never used xUnit, but I doubt its very different. I will start asap.

**Comment by galvesribeiro** [comment]
For whatever reason GH is now closing issues when you merge stuff. Re-opening this as #235 partially complete this one.

**Comment by jterry75** [comment]
This has one sample under `test/Docker.DotNet.Tests` to get started.
